### PR TITLE
Modifying to support S3 additional metrics

### PR DESCRIPTION
**Pull request by TheM0ng00se**

### CODE REVIEW

File: cinq_auditor_required_tags/__init__.py [hunk 1]
```diff
@@ -326,30 +331,38 @@ def process_actions(self, actions):
...
-                    metrics = {}
+                metrics = {}
+                self.log.info('Resource is {} {}'.format(resource, dir(resource)))
+                if resource.resource_type_id == AWS_EC2_INSTANCE:
+                    for prop in resource.properties:
+                        if prop.name == "instance_type":
+                            instance_type = prop.value
+                        if prop.name == "public_ip":
+                            public_ip = prop.value
+                    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```
Comment (rnikoopour) [inline cinq_auditor_required_tags/__init__.py]: `instance_type` and `public_ip` have the possibility to not be set.

I would change 
```
if resource.resource_type_id == AWS_EC2_INSTANCE:
    for prop in resource.properties:
        if prop.name == "instance_type":
            instance_type = prop.value
        if prop.name == "public_ip":
            public_ip = prop.value
            
    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```
to
```
if resource.resource_type_id == AWS_EC2_INSTANCE:
    instance_type = "No instance type found"
    public_ip = "No public ip found"
    for prop in resource.properties:
        if prop.name == "instance_type":
            instance_type = prop.value
        if prop.name == "public_ip":
            public_ip = prop.value
            
    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```

File: cinq_auditor_required_tags/__init__.py [hunk 2]
```diff
@@ -326,30 +331,38 @@ def process_actions(self, actions):
...
+
+                elif resource.resource_type_id == AWS_S3_BUCKET:
+                    for prop in resource.properties:
+                        if prop.name == "metrics":
+                            metrics = prop.value
```
Comment (rnikoopour) [inline cinq_auditor_required_tags/__init__.py]: Same kind of thing here.  It's possible that `metrics` is not going to be set and you reference further below.

File: cinq_auditor_required_tags/__init__.py (hunk 1 above)
Comment (TheM0ng00se) [reply cinq_auditor_required_tags/__init__.py]: will change to a generic "Not Found"

File: cinq_auditor_required_tags/__init__.py (hunk 2 above)
Comment (TheM0ng00se) [reply cinq_auditor_required_tags/__init__.py]: I'll set to 'unavailable'. Because 0 and 0 are valid values.

**Pull request merged**